### DISCUSSION TITLE
How do specify properties using [Reactive], that are using my property?

### DISCUSSION THREAD

**Discussion by janniksam** [reactiveui/ReactiveUI#3299] [Q&A] [answered]
For example having this in my ViewModel:

```cs
[Reactive]public string FirstName { get; set; }
[Reactive]public string LastName { get; set; }

public string FullName => FirstName + " " + LastName;
```

Is there a way to automatically get INotifyPropertyChanged invoked for FullName, whenever one of the first two properties are changed?
In other words, is there a way to define additional properties, that are weaved into the first two properties?

**Comment by anaisbetts** [comment, marked as answer]
Yes, you are looking for `ToProperty` and `ObservableAsPropertyHelper` - https://www.reactiveui.net/docs/handbook/observable-as-property-helper/

**Comment by glennawatson** [comment]
There's also another built in functionality in the fody for this https://github.com/reactiveui/ReactiveUI/blob/4894cae73e6070908eeaefd7d70b58d315086e74/src/ReactiveUI.Fody.Tests/Mocks/FacadeModel.cs#L40
